src/gui/accountmanager.cpp

```diff
@@ -78,40 +77,45 @@ bool AccountManager::restore()
 bool AccountManager::restoreFromLegacySettings()
 {
     qCInfo(lcAccountManager) << "Migrate: restoreFromLegacySettings, checking settings group"
-             << Theme::instance()->appName();
+                             << Theme::instance()->appName();
```

Comment:
@jturcotte I think token alignement looks nice and clean but breaks easily and is annoying to reformat. If I aligned this manually, I'd not do token alignment because `qCInfo(lcAccountManager)` is too long for that. 

What if we did `AlwaysBreak`? Gain neatness at the expense of shorter lines and more vertical space used.